src/kOS/Suffixed/Part/DockingPortValue.cs

```diff
@@ -18,9 +18,12 @@ public DockingPortValue(ModuleDockingNode module, SharedObjects sharedObj) : bas
 
         private void DockingInitializeSuffixes()
         {
-            AddSuffix("AQUIRERANGE", new Suffix<float>(() => module.acquireRange));
-            AddSuffix("AQUIREFORCE", new Suffix<float>(() => module.acquireForce));
-            AddSuffix("AQUIRETORQUE", new Suffix<float>(() => module.acquireTorque));
+            AddSuffix("AQUIRERANGE", new Suffix<float>(() => { throw new kOS.Safe.Exceptions.KOSDeprecationException("18.0", "AQUIRERANGE", "ACQUIRERANGE", string.Empty); }));
+            AddSuffix("AQUIREFORCE", new Suffix<float>(() => { throw new kOS.Safe.Exceptions.KOSDeprecationException("18.0", "AQUIREFORCE", "ACQUIREFORCE", string.Empty); }));
+            AddSuffix("AQUIRETORQUE", new Suffix<float>(() => { throw new kOS.Safe.Exceptions.KOSDeprecationException("18.0", "AQUIRETORQUE", "ACQUIRETORQUE", string.Empty); }));
```

Comment:
I won't be in a position to edit code until tomorrow morning.  I can do it then or you can do it before if you like.